scripts/projectUpdater.ps1

```diff
@@ -1,10 +1,6 @@
 
 param()
 
-# include
-. "$env:HOME/.apollox/scripts/utils/formatJson.ps1"
```

Comment:
instead check if the file exists, then include then

Reply:
Those files were removed from this repo on TOS7, so this files will never exist.

Reply:
Can occur if the user change the repo, branch, tag. Is safer check if they exists then include.

Reply:
In this cases it may happen, I will add the check

Reply:
Added